### ISSUE TITLE
Create Empty Script Dialog - Script Name Field does not have Focus 

### ISSUE THREAD

**Issue by Phuket2** [omz/Pythonista-Issues#432]
When you create a new script, you have to tap the field to enter the name as i does not have focus.  If you select create UIFile, that dialog behaves correctly
311014

**Comment by omz** [comment]
Hmm, I can't reproduce this, and I don't see an obvious reason why the behavior should differ for different templates... :/

**Comment by Phuket2** [comment]
I think it maybe a HW keyboard thing. I will check later and get back. I had just tested again, it neither worked....

**Comment by Phuket2** [comment]
@omz, I am going to close this item.  I can see without the HW keyboard attached it sort of works the way it should. As you have to click the field to invoke the keyboard. With a HW keyboard, it would make more sense that the field has focus.  I would say there are a lot more pressining items than this. It is strange, sometimes when I select new UIFile the field does have focus. But mostly not. I could not figure out how to reproduce consistently. Oh, well it's a small refinement issue for another day

**Comment by omz** [comment]
Now that I think about it again, it probably doesn't have to do with the kind of keyboard you're using, but rather with the amount of files you have in your library (see #405).

I don't have a lot of data on my test device, so the folder picker dialog appears relatively quickly (and I can't reproduce this issue at all), but if you have a lot, the delay might somehow confuse the keyboard as well...

**Comment by Phuket2** [comment]
@omz , that makes sense. I went to another iPad were I dont have a lot on it and the keyboard popped up and the field had focus. I dont have many files on that Ipad also.  I didn't try it with HW keyboard, as I would have had to dig up a Bluetooth keyboard. I am using the Apple Magic Keyboard.
But without being critical, it does appear Pythonista's file system has its limitations.  Eg, in working copy I cloned [ActiveState/code](https://github.com/ActiveState/code) recipes.  Then i dragged and dropped the Python dir onto This IPad. At first I thought nothing happened. Anyway, it did actually copy the dir(insane number of folders and files).  Then I had a few crashes, then my screen blanked out.  The only way to get the iPad Pro working again was to do a hard reset(power switch + Home).  
Again, not being critical.  I am just pushing too hard. I just in my wisdom thought it would be nice to have them there so i could just go through them and run the recipes that would work in Pythonista with the idea of deleting the folder at some later point.  
Anyway, its just more info than anything else.

***It does bring up one question though.  I deleted that huge folder as soon as I could get back into the app.  But can this create a type of fragmentation in your indexes?  If so, is there a way force Pythonista to rebuild indexes that may get fragmented from large amounts of file adds/deletes?***
